library/choose_disk.py

```diff
@@ -361,6 +363,17 @@ def disk_label(partition):
     if "ceph journal" in stdout:
         return "journal"
 
+    # Then, let's search for metadata coming from ceph-volume
+    output_cmd = subprocess.Popen("ceph-volume lvm list --format=json {}".format(partition),
+                                  shell=True, stdout=subprocess.PIPE)
+    raw_json, _ = output_cmd.communicate()
+    json = json.loads(raw_json)
+    for item in json:
+        current_item = json[item]
```

Comment:
If you need to look at values as well as the keys from the JSON, you should iterate over `items()` that will give you both, instead of asking the dictionary for `current_item`.

That would look like:

    for key, value in json.items():

Reply:
Fixed it, thx.